scripts/docker/webui/base/Dockerfile

```diff
@@ -3,11 +3,17 @@
 FROM ubuntu:16.04
```

Comment:
A general note regarding the Dockerfile: there is a readme in `scripts/docker/webui/README.md`.

The problem has probably been there before but there is an [error in the docs](https://github.com/ElektraInitiative/libelektra/blame/master/scripts/docker/webui/README.md#L15). The build context should likely be the libelektra root. The how-to from the docs does not work as-is.

Further we should probably increment the elektra web release version as you are introducing some major changes.